bin/make/gemuse-libs.sh

```diff
@@ -73,7 +73,7 @@ function fat_lib_with_lipo {
   xcrun lipo -create "${arm_lib}" "${sim_lib}" \
     -output "${fat_lib}"
 
-  "${HOME}/.calabash/calabash-codesign/apple/resign-dylib.sh" \
+  "${HOME}/.test-cloud-dev/test-cloud-dev-ios-keychain/apple/resign-dylib.sh" \
```

Comment:
This one is hard. :/

The resign-dylib.sh script no longer exists.

We need to replace this with a codesign command.

Reply:
I have reverted this change for now.
Should I take care about replacement in scope of this PR and this work item?
If yes - could you please share some info about `resign-dylib.sh` since I don't have context about purpose of this script and i can't find it in repo